src/highdicom/utils.py

```diff
@@ -307,9 +319,7 @@ def map_pixel_into_coordinate_system(
 
     column_offset = float(coordinate[0])
     row_offset = float(coordinate[1])
-    pixel_matrix_coordinate = np.array([
-        [row_offset, column_offset, 0.0, 1.0]
-    ])
+    pixel_matrix_coordinate = np.array([[column_offset, row_offset, 0.0, 1.0]])
```

Comment:
Thanks for the clarification. Coordinates in the slide coordinate system shall not be negative, but I forgot that this may occur in the patient coordinate system. I have removed the check.

Addressed with e67edb963fbf3ddb7bb261760768f279373ae068

Reply:
@CPBridge I refactored that quite a bit (see 6f543d476888a73fb8e710c97e8d1a2240f4a1ac) to facilitate reuse of the affine transformation matrix. In addition, I implemented the inverse transformation. It passes all the tests, but it would be great if you could test the inverse mapping (from frame of reference into pixel matrix) on a few more radiology images.